sycl/test/abi/sycl_symbols_linux.dump

```diff
@@ -3533,4 +3533,9 @@ _ZNK2cl4sycl9exception11has_contextEv
 _ZNK2cl4sycl9exception4whatEv
 __sycl_register_lib
 __sycl_unregister_lib
+_ZN2cl4sycl6opencl12make_programERKNS0_7contextEm
+_ZN2cl4sycl6opencl13make_platformEm
+_ZN2cl4sycl6opencl11make_deviceEm
+_ZN2cl4sycl6opencl10make_queueERKNS0_7contextEm
+_ZNK2cl4sycl8platform9getNativeEv
```

Comment:
The script sorts these entries. Could you please put new definitions to appropriate places (or re-generate it with the script), so that we could avoid large diffs in future?

Reply:
How is this sorted? And what is the script to re-generate?
The following shows unexpectedly many diffs:
`$ cat sycl_symbols_linux.dump | sort | diff sycl_symbols_linux.dump -`

Reply:
@smaslov-intel Here's the link to instruction: https://github.com/intel/llvm/blob/sycl/sycl/doc/ABIPolicyGuide.md#automated-abi-changes-testing

Reply:
Thanks. It is inconvenient that after re-generating you need to put back the LIT commands. Also there were other diffs except mine, so if sorted order is not somehow enforced it will be broken most of the time. Why don't just allow adding to the end?

Reply:
@smaslov-intel thanks for feedback. Adding LIT commands is truly inconvenient, I'll see if I can improve it. The reason for keeping it sorted is to avoid large diffs when big changes come. But it seems not everyone follows official guide, so we still get larger diffs sometimes. That's also something to work on.